### PR TITLE
add log when unable to resolve uid by xref service.

### PR DESCRIPTION
**Pull request by vwxyzh**

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.Engine/XrefServiceResolver.cs [hunk 1]
```diff
@@ -77,6 +77,7 @@ public XrefServiceResolver(HttpClient client, ImmutableArray<string> xrefService
                     return value[0];
                 }
             }
+            Logger.LogVerbose($"Cannot resolve uid({uid}).");
```
Comment (vicancy) [inline src/Microsoft.DocAsCode.Build.Engine/XrefServiceResolver.cs]: Aggregate unresolved uids?

File: src/Microsoft.DocAsCode.Build.Engine/XrefServiceResolver.cs [hunk 2]
```diff
@@ -55,6 +56,19 @@ public XrefServiceResolver(HttpClient client, ImmutableArray<string> xrefService
                     externalXRefSpec.AddOrUpdate(tuple.Item1, tuple.Item2, (s, x) => x + tuple.Item2);
                 }
             }
+            if (unresolvedUidList.Count > 0 && Logger.LogLevelThreshold <= LogLevel.Verbose)
+            {
+                var sb = new StringBuilder("Cannot resolve  uids by xref service:", 100 + 64 * (Math.Min(100, unresolvedUidList.Count)));
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.Engine/XrefServiceResolver.cs]: >   [](start = 58, length = 2)

an extra space here

File: src/Microsoft.DocAsCode.Build.Engine/XrefServiceResolver.cs [hunk 3]
```diff
@@ -55,6 +56,19 @@ public XrefServiceResolver(HttpClient client, ImmutableArray<string> xrefService
...
+                sb.Append("Cannot resolve ");
+                sb.Append(unresolvedUidList.Count);
+                sb.Append(" uids by xref service, top 100:");
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.Engine/XrefServiceResolver.cs]: >100 [](start = 54, length = 3)

min(100, unresolvedUidList.Count) ?

File: src/Microsoft.DocAsCode.Build.Engine/XrefServiceResolver.cs (hunk 2 above)
Comment (vicancy) [inline src/Microsoft.DocAsCode.Build.Engine/XrefServiceResolver.cs]: Also the endpoint value for xref service?

**Pull request merged**